docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+services:
+  app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: 'python3 manage.py runserver 0.0.0.0:8000 --noreload'
+    ports:
+      - '8000:8000'
+    volumes:
+      - ./:/app
+    env_file:
+      - .env
+    environment:
+      - PYTHONUNBUFFERED=1
```

Comment:
make sure that static and media files are persistent while creating and removing docker container.